AUTHORS

```diff
@@ -214,3 +214,4 @@ Jim Cai <jimcai@stanford.edu>
 Richard Moch <richard.moch@gmail.com>
 Randy Ostler <rando305@gmail.com>
 Thomas Young <thomas@upcoder.com>
+Christopher Lee <christopherlee2012@gmail.com>
```

Comment:
I can't believe I'm actually making a comment on an AUTHORS change, but please use your edX email address so people can tell from the list what your affiliation is. :-P